meta/main.yml

```diff
@@ -2,20 +2,20 @@
 galaxy_info:
   author: Nick Janetakis
   description: The official rvm role to install and manage your ruby versions.
-  company:
+  company: ''
   license: license (MIT)
   min_ansible_version: 2.2
 
   platforms:
-  - name: EL
-    versions:
-    - all
-  - name: Ubuntu
-    versions:
-    - all
+    - name: EL
```

Comment:
According the doc this intentation is not necessary.
https://galaxy.ansible.com/intro#meta